lib/repositories/garageRepository.dart

```diff
@@ -24,54 +24,54 @@ class GarageRepository {
 
   Future<List<Garage>> getGarages() {
     return Future(
-      () async {
```

Comment:
This file should not have been modified

Reply:
This should be the exact same besides formatting. Once I fix the other issue I'll try pulling down this file individually from master.